Angular Compiler requires TypeScript >=3.4.0 and <3.6.0 but 3.7.5 was found instead.

ERROR in The Angular Compiler requires TypeScript >=3.4.0 and <3.6.0 but 3.7.5 was found instead.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! website@0.0.0 build: `ng build "--prod" "--base-href" "./index.html" "--progress" "false"`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the website@0.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-02-04T18_56_16_050Z-debug.log

--- a/multi-cloud-ai-workflow-az/website/package.json
+++ b/multi-cloud-ai-workflow-az/website/package.json
@@ -56,6 +56,6 @@
         "protractor": "^5.4.2",
         "ts-node": "^8.5.4",
         "tslint": "^5.20.1",
-        "typescript": "^3.5.3"
+        "typescript": "3.5.3"
     }
 }

